lib/pages/farm/desktop_farm.dart

```diff
@@ -125,7 +125,7 @@ class _DesktopFarmState extends State<DesktopFarm> {
             SizedBox(
               //contains list of allfarms cards
               width: layoutWdt,
-              height: layoutHgt,
+              height: layoutHgt - 120,
```

Comment:
I would always advise against magic numbers ( what does 120 represent? )
But the usage in this class is too broad to have it refactored in the scope of this PR.